sdk/src/client/stronghold/mod.rs

```diff
@@ -554,7 +555,7 @@ mod tests {
 
         let stronghold_path = "test_clear_key.stronghold";
         let mut adapter = StrongholdAdapter::builder()
-            .password("drowssap")
```

Comment:
never noticed this, what does it stand for :D

Reply:
reversed?

Reply:
it's bad english for "drowning in sap/juice" :yum: :trollface: